weblate/urls.py

```diff
@@ -186,6 +186,12 @@
         weblate.lang.views.CreateLanguageView.as_view(),
         name="create-language",
     ),
+    path(
+        # Site-wide add-ons
+        "addons/",
```

Comment:
This should probably go into manage namespace:

```suggestion
        "manage/addons/",
```